lib/telnyx/network.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Telnyx
+  class Network < APIResource
+    include Telnyx::APIOperations::Save
```

Comment:
Do we need delete, save & update methods? as in excel we only need show and index